pkg/eventprocessor/impl/eventprocessor_test.go

```diff
@@ -293,17 +293,20 @@ type contractCalls struct {
 	transfer      contractTransferFromSender
 }
 
-type dbReader func(string) []int64
-type contractRunSQLBlockSender func([]string) []common.Hash
-type contractCreateTableSender func(string) common.Hash
-type contractSetControllerSender func(controller common.Address) common.Hash
-type contractTransferFromSender func(controller common.Address) common.Hash
-type checkReceipts func(*testing.T, ...eventprocessor.Receipt) func() bool
+type (
+	dbReader                    func(string) []int64
+	contractRunSQLBlockSender   func([]string) []common.Hash
+	contractCreateTableSender   func(string) common.Hash
+	contractSetControllerSender func(controller common.Address) common.Hash
+	contractTransferFromSender  func(controller common.Address) common.Hash
+	checkReceipts               func(*testing.T, ...eventprocessor.Receipt) func() bool
+)
```

Comment:
Started using `gofumpt` formatter instead of `gopls`. Apparently in includes all `gopls` but is stricter and does extra stuff like this. So far I like it.

Reply:
Good question. Looking. Indeed it can https://golangci-lint.run/usage/linters/#gofumpt